SPECS/kata-containers-cc/kata-containers-cc.spec

```diff
@@ -0,0 +1,281 @@
+install -D -m 0644 versions.yaml        %{buildroot}%{osbuilder}/versions.yaml
+cp -aR tools/osbuilder/rootfs-builder   %{buildroot}%{osbuilder}/tools/osbuilder
+cp -aR tools/osbuilder/initrd-builder   %{buildroot}%{osbuilder}/tools/osbuilder
+cp -aR tools/osbuilder/scripts          %{buildroot}%{osbuilder}/tools/osbuilder
+popd
+
+# Symlinks for cc binaries
+mkdir -p %{buildroot}%{coco_bin}
+mkdir -p %{buildroot}%{share_kata}
+mkdir -p %{buildroot}%{coco_path}/libexec
+mkdir -p %{buildroot}/etc/systemd/system/containerd.service.d/
+ln -s /usr/bin/cloud-hypervisor               %{buildroot}%{coco_bin}/cloud-hypervisor
+ln -s /usr/bin/containerd                     %{buildroot}%{coco_bin}/containerd
+ln -s /usr/share/cloud-hypervisor/vmlinux.bin %{buildroot}%{share_kata}/vmlinux.container
+
+ln -sf /usr/libexec/virtiofsd %{buildroot}/%{coco_path}/libexec/virtiofsd
+install -D -m 0644 %{SOURCE3} %{buildroot}/etc/systemd/system/containerd.service.d/containerd-for-cc-override.conf
+
+find %{buildroot}/etc
+
+# Agent
+pushd %{_builddir}/%{name}-%{version}/src/agent
+
+mkdir -p %{buildroot}%{osbuilder}/src/agent/samples/policy
+cp -aR samples/policy/all-allowed         %{buildroot}%{osbuilder}/src/agent/samples/policy
+install -D -m 0755 mount_tar.sh           %{buildroot}%{osbuilder}/src/agent/mount_tar.sh
+install -D -m 0755 kata-containers.target %{buildroot}%{osbuilder}/kata-containers.target
+install -D -m 0755 kata-agent.service.in  %{buildroot}%{osbuilder}/kata-agent.service.in
+install -D -m 0755 coco-opa.service       %{buildroot}%{osbuilder}/coco-opa.service
+install -D -m 0755 target/x86_64-unknown-linux-gnu/release/kata-agent %{buildroot}%{osbuilder}/kata-agent
+
+popd 
+
+# Runtime
+pushd %{_builddir}/%{name}-%{version}/src/runtime
+install -D -m 0755 containerd-shim-kata-v2 %{buildroot}/usr/local/bin/containerd-shim-kata-cc-v2
+install -D -m 0755 kata-monitor %{buildroot}%{coco_bin}/kata-monitor
+install -D -m 0755 kata-runtime %{buildroot}%{coco_bin}/kata-runtime
+install -D -m 0755 data/kata-collect-data.sh %{buildroot}%{coco_bin}/kata-collect-data.sh
+
+install -D -m 0644 config/configuration-clh.toml %{buildroot}/%{defaults_kata}/configuration-clh.toml
+ln -s configuration-clh.toml %{buildroot}/%{defaults_kata}/configuration.toml
+sed -i 's|/usr|/opt/confidential-containers|g' %{buildroot}/%{defaults_kata}/configuration-clh.toml
+popd
+
+# Tardev-snapshotter
+pushd %{_builddir}/%{name}-%{version}/src/tardev-snapshotter/
+sed -i -e 's/containerd.service/kubelet.service/g' tardev-snapshotter.service
+install -m 0644 -D -t %{buildroot}%{_unitdir} tardev-snapshotter.service
+install -D -m 0755 target/release/tardev-snapshotter  %{buildroot}/usr/bin/tardev-snapshotter
+popd
+
+# The UVM rootfs will be used to build an img file at the first host boot.
+install -D -m 0755 %{_builddir}/%{name}-%{version}/tools/osbuilder/image-builder/image_builder.sh   %{buildroot}%{osbuilder}/tools/osbuilder/image-builder/image_builder.sh
```

Comment:
install section for tools?

Reply:
I think this is necessary in the %files section instead of the %install section